[debug] [2025-04-23T21:13:58.607Z] ----------------------------------------------------------------------
[debug] [2025-04-23T21:13:58.612Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\vedaa\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js projects:list --json
[debug] [2025-04-23T21:13:58.613Z] CLI Version:   10.3.0
[debug] [2025-04-23T21:13:58.613Z] Platform:      win32
[debug] [2025-04-23T21:13:58.613Z] Node Version:  v22.14.0
[debug] [2025-04-23T21:13:58.614Z] Time:          Wed Apr 23 2025 14:13:58 GMT-0700 (Pacific Daylight Time)
[debug] [2025-04-23T21:13:58.614Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-04-23T21:13:58.626Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-04-23T21:13:58.626Z] > authorizing via signed-in user (veda.aarushi@gmail.com)
[debug] [2025-04-23T21:13:58.631Z] > refreshing access token with scopes: []
[debug] [2025-04-23T21:13:58.632Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[debug] [2025-04-23T21:13:58.814Z] <<< HTTP RESPONSE 400 {"date":"Wed, 23 Apr 2025 21:14:00 GMT","pragma":"no-cache","cache-control":"no-cache, no-store, max-age=0, must-revalidate","expires":"Mon, 01 Jan 1990 00:00:00 GMT","content-type":"application/json; charset=utf-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","server":"scaffolding on HTTPServer2","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3=\":443\"; ma=2592000,h3-29=\":443\"; ma=2592000","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2025-04-23T21:13:58.818Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-04-23T21:13:58.934Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 401
[debug] [2025-04-23T21:13:58.934Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-04-23T21:13:58.936Z] HTTP Error: 401, Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.
[debug] [2025-04-23T21:13:58.999Z] FirebaseError: HTTP Error: 401, Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.
    at module.exports (C:\Users\vedaa\AppData\Roaming\npm\node_modules\firebase-tools\lib\responseToError.js:47:12)
    at RetryOperation._fn (C:\Users\vedaa\AppData\Roaming\npm\node_modules\firebase-tools\lib\apiv2.js:286:39)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
[error] 
[error] Error: Failed to list Firebase projects. See firebase-debug.log for more info.
[debug] [2025-04-23T21:14:59.318Z] ----------------------------------------------------------------------
[debug] [2025-04-23T21:14:59.324Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\vedaa\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js projects:create my-updated-flashcard --json
[debug] [2025-04-23T21:14:59.324Z] CLI Version:   10.3.0
[debug] [2025-04-23T21:14:59.325Z] Platform:      win32
[debug] [2025-04-23T21:14:59.325Z] Node Version:  v22.14.0
[debug] [2025-04-23T21:14:59.325Z] Time:          Wed Apr 23 2025 14:14:59 GMT-0700 (Pacific Daylight Time)
[debug] [2025-04-23T21:14:59.326Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-04-23T21:14:59.337Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-04-23T21:14:59.338Z] > authorizing via signed-in user (veda.aarushi@gmail.com)
[debug] [2025-04-23T21:14:59.343Z] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[debug] [2025-04-23T21:14:59.344Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[debug] [2025-04-23T21:14:59.500Z] <<< HTTP RESPONSE 400 {"pragma":"no-cache","date":"Wed, 23 Apr 2025 21:15:00 GMT","cache-control":"no-cache, no-store, max-age=0, must-revalidate","expires":"Mon, 01 Jan 1990 00:00:00 GMT","content-type":"application/json; charset=utf-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","server":"scaffolding on HTTPServer2","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3=\":443\"; ma=2592000,h3-29=\":443\"; ma=2592000","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2025-04-23T21:14:59.503Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects  
 {"projectId":"my-updated-flashcard","name":"my-updated-flashcard"}
[debug] [2025-04-23T21:14:59.580Z] <<< HTTP RESPONSE 401 {"www-authenticate":"Bearer realm=\"https://accounts.google.com/\", error=\"invalid_token\"","vary":"X-Origin, Referer, Origin,Accept-Encoding","content-type":"application/json; charset=UTF-8","date":"Wed, 23 Apr 2025 21:15:00 GMT","server":"ESF","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","server-timing":"gfet4t7; dur=16","alt-svc":"h3=\":443\"; ma=2592000,h3-29=\":443\"; ma=2592000","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2025-04-23T21:14:59.580Z] <<< HTTP RESPONSE BODY {"error":{"code":401,"message":"Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.","status":"UNAUTHENTICATED","details":[{"@type":"type.googleapis.com/google.rpc.ErrorInfo","reason":"ACCESS_TOKEN_TYPE_UNSUPPORTED","metadata":{"method":"google.cloudresourcemanager.v1.Projects.CreateProject","service":"cloudresourcemanager.googleapis.com"}}]}}
[debug] [2025-04-23T21:14:59.656Z] FirebaseError: HTTP Error: 401, Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.
    at module.exports (C:\Users\vedaa\AppData\Roaming\npm\node_modules\firebase-tools\lib\responseToError.js:47:12)
    at Request._callback (C:\Users\vedaa\AppData\Roaming\npm\node_modules\firebase-tools\lib\api.js:41:35)
    at self.callback (C:\Users\vedaa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:185:22)
    at Request.emit (node:events:518:28)
    at Request.emit (node:domain:489:12)
    at Request.<anonymous> (C:\Users\vedaa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:1154:10)
    at Request.emit (node:events:518:28)
    at Request.emit (node:domain:489:12)
    at IncomingMessage.<anonymous> (C:\Users\vedaa\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:1076:12)
    at Object.onceWrapper (node:events:632:28)
    at IncomingMessage.emit (node:events:530:35)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1698:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)
[error] 
[error] Error: Failed to create project. See firebase-debug.log for more info.
